src/locales/si/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,565 @@
+# Sinhala translations for PROJECT.
```

Comment:
@SrilalS in another PR try to update the begining of this file, it should be:

```
# Copyright (c) Facebook, Inc. and its affiliates.
#
# This source code is licensed under the MIT license found in the
# LICENSE file in the root directory of this source tree.
```

Reply:
@elinguiuriel I have made the changes. 
#90